### PR TITLE
2.majas_darbs_Dzintars

### PR DESCRIPTION
**Pull request by dzintarsm**
First part of homework done. Some known issues and improvements still exist.
Known stuff:

- Only the 1st part of homework done (no collection creation)
- Missing the part where newly created project is being set as active, so currently the environments are set up for the first project for this account
- Need to improve file structure (using the cheap one)
- Issue with error in console when being run two times in a row (not sure if worth spending time to investigate).

### CODE REVIEW

File: features/step_definitions/2.majas_darbs_steps.rb [hunk 1]
```diff
@@ -0,0 +1,150 @@
+require 'rest-client'
+require 'test-unit'
+
+When(/^I create a new project$/) do
+    login_positive #first login to generate a cookie
+    @test_user.session_cookie.to_json
+
+    project_name = (0...10).map { (65 + rand(26)).chr }.join #generate random project name
```
Comment (aakacenoks) [inline features/step_definitions/2.majas_darbs_steps.rb:8]: Good idea, I wonder why I didn't think of that

File: features/step_definitions/2.majas_darbs_steps.rb [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+    project_payload = {
+        :name => project_name,
+        :type => "basic"}.to_json
+    response = RestClient::Request.execute(method: :post,
```
Comment (aakacenoks) [inline features/step_definitions/2.majas_darbs_steps.rb:13]: I guess you could create the api helper file that we used for post, get and other methods

**Review by aakacenoks — APPROVED** [review]
Approved with some comments

**Comment by usilins** [comment]
Checked

**Pull request merged**